share/functions/__fish_generate_password.fish

```diff
@@ -0,0 +1,4 @@
+function __fish_generate_password --description 'Generate password'
```

Comment:
Please do not include this function. It does not generate cryptographically-random passwords, and passwords should not be passed on the command-line anyway (I understand that some Windows programs accept them).